website/src/parsers/js/typescript-eslint-parser.js

```diff
@@ -0,0 +1,41 @@
+import defaultParserInterface from './utils/defaultESTreeParserInterface';
+import pkg from 'typescript-eslint-parser/package.json';
+
+const ID = 'typescript-eslint-parser';
+const name = 'typescript-eslint-parser';
```

Comment:
You can just use `displayName: ID`, it's fine when they match (we do this in other parsers too).